From 44d9d58461e15314edf507d0379bff991d7e9964 Mon Sep 17 00:00:00 2001
From: Ihar Hrachyshka <ihar.hrachyshka@gmail.com>
Date: Thu, 11 Sep 2025 23:40:51 -0400
Subject: [PATCH ovn 2/2] tests: Use localhost when setting "wrong" ovn-remote.

In some isolated environments (e.g. in nixpkgs build sandbox), the
network namespace doesn't have any routes or interfaces but `lo`. In
this case, an attempt to connect to 192.168.0.10 results in ENETUNREACH,
producing an unexpected "Network unreachable" warning message in service
log file - breaking the test cleanup checks.

Since the test case doesn't seem to care if the address is available, as
long as there is no SB database actually running at the configured
ovn-remote port, use the localhost address instead (which is always
present, even in the most isolated environments).

Signed-off-by: Ihar Hrachyshka <ihar.hrachyshka@gmail.com>
---
 tests/ovn-controller.at | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/ovn-controller.at b/tests/ovn-controller.at
index bec06e527..a8a9a2da2 100644
--- a/tests/ovn-controller.at
+++ b/tests/ovn-controller.at
@@ -404,7 +404,7 @@ check_sbdb_connection () {
 
 OVS_WAIT_UNTIL([check_sbdb_connection connected])
 
-ovs-vsctl set open . external_ids:ovn-remote=tcp:192.168.0.10:6642
+ovs-vsctl set open . external_ids:ovn-remote=tcp:127.0.0.1:12345
 OVS_WAIT_UNTIL([check_sbdb_connection 'not connected'])
 
 # reset the remote for clean-up
-- 
2.50.1

